docs/configuration.md

```diff
@@ -1141,6 +1141,48 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.scheduler.blacklist.enabled</code></td>
+  <td>true</td>
+  <td>
+    If set to ture, executor blacklist feature will be enabled to avoid allocte new task on bad executor. The logic to define bad executor is based on BlacklistStrategy which is also configurable.
```

Comment:
A few typos here (ture, allocte), but I'd also like to suggest a rewording:

If set to "true", prevent Spark from scheduling tasks on executors that have been blacklisted due to too many task failures. The blacklisting algorithm can be further controlled by the other "spark.scheduler.blacklist" configuration options.